tests/sentry/seer/anomaly_detection/test_store_data.py

```diff
@@ -124,3 +125,29 @@ def test_anomaly_detection_fetch_historical_data_crash_rate_alert(self):
         assert result
         assert self.time_1 in result.data.get("data").get("intervals")
         assert 1 in result.data.get("data").get("groups")[0].get("series").get("sum(session)")
+
+    def test_anomaly_detection_fetch_historical_data_apdex_alert(self):
+        # TODO generate whatever apdex data is
+        # store a transaction event
+        # apply the apdex formula(?)
+        event_data = load_data("transaction")
+        event_data["timestamp"] = iso_format(self.time_1_dt)
+        event_data["start_timestamp"] = iso_format(self.time_1_dt)
+        event_data["event_id"] = "a" * 32
+        self.store_event(data=event_data, project_id=self.project.id)
+
+        alert_rule = self.create_alert_rule(
+            projects=[self.project],
+            dataset=Dataset.PerformanceMetrics,
+            name="JustAValidRule",
+            query="",
+            aggregate="apdex(300)",
+            time_window=1,
+            threshold_type=AlertRuleThresholdType.BELOW,
+            threshold_period=1,
+        )
+        snuba_query = SnubaQuery.objects.get(id=alert_rule.snuba_query_id)
+        result = fetch_historical_data(alert_rule, snuba_query, self.project)
+        assert result
+        assert {"time": int(self.time_1_ts), "count": 1} in result.data.get("data")
```

Comment:
I haven't gotten this to pass yet as I'm not sure what I need to store in order for a `count` to show up in the results